src/transformers/models/upernet/modeling_upernet.py

```diff
@@ -301,6 +301,12 @@ class UperNetPreTrainedModel(PreTrainedModel):
     main_input_name = "pixel_values"
     supports_gradient_checkpointing = True
 
+    def _init_weights(self, module):
```

Comment:
`_init_weights` is now required to use `from_pretrained` on a PreTrainedModel. This model did not hve it.